src/ocean/text/convert/Integer_tango.d

```diff
@@ -291,9 +303,18 @@ private struct _FormatterInfo(T)
     byte    radix;
     T[]     prefix;
     T[]     numbers;
+    DecimalMark decimal_mark;
 }
 
-Const!(T)[] formatter(T, N) (T[] dst, N i_, char type, char pre, int width)
+private struct DecimalMark
```

Comment:
Hm am I understanding correctly that you're adding this feature to the old tango formatter too? (That seems unnecessary, if that's the case.)